public/_autoload.php

```diff
@@ -32,13 +35,13 @@ function resolve_path($name)
   return file_exists($publicRoot) ? realpath($publicRoot) : realpath($appRoot);
 }
 
-function __autoload($class_name)
-{
+
+spl_autoload_register(function ($class_name) {
```

Comment:
ahh, nm, the It looked like sp1_autoload_register was inside the "function __autoload"